Lib/test/test_inspect/test_inspect.py

```diff
@@ -4027,44 +4027,217 @@ def __init__(self, b):
                            ('bar', 2, ..., "keyword_only")),
                           ...))
 
-    def test_signature_on_class_with_decorated_new(self):
+    def test_signature_on_class_with_decorated_init(self):
         def identity(func):
```

Comment:
Since `identity()` and `Desc` are the same in both tests, why not move them at global level?